base_contact/README.rst

```diff
@@ -0,0 +1,7 @@
+This module allows you to manage your contacts
+==============================================
+
+Features
+--------
+    - a contact can have several job positions in different companies,
+    - birthdate and nationality can be defined.
```

Comment:
This porting would be an opportunity to use OCA readme template:
https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst